test/test.js

```diff
@@ -1,7 +1,27 @@
 const test = require('tape');
+const request = require('supertest');
+const app = require('../src/app');
+const dbRun = require('../src/database/build');
 
-test('testing the tape', (assert) => {
-  const num = 1;
-  assert.equal(num, 1, 'pass');
-  assert.end();
+test('Request signup route ', (t) => {
+  dbRun((err) => {
+    if (err) t.error('error', err);
+    request(app)
+      .post('/signup')
+      .send({
+        userName: 'ssadrgvd',
+        password: '123456',
+        confirmPassword: '123456',
+      })
+      .expect(200)
+      .expect('Content-Type', 'text/html; charset=utf-8')
+      .end((errr, res) => {
+        if (err) {
+          t.error(errr);
+        } else {
+          t.equal(JSON.parse(res.text).msg, 'تم انشاء الحساب بنجاح، يمكنكي الآن تسجيل الدخول', 'must return msg of sucsses signup');
+          t.end();
+        }
+      });
+  });
```

Comment:
@AymanAlqoqa Looks good!

Reply:
what you suggest for this!

Reply:
No suggestion, I was just saying it was a good test :)

Reply:
thanks a lot :)